char allocation_mode
bool greedorprobability
char allocation_method

allocation_robot_info[] all_allocation_robot_info
allocation_task_info[] all_allocation_task_info                                 
